datastore/spec/quickstart_spec.rb

```diff
@@ -1,55 +0,0 @@
-# Copyright 2016 Google, Inc
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-require "rspec"
-require "rspec/retry"
-require "google/cloud/datastore"
-
-RSpec.configure do |config|
-  # show retry status in spec process
-  config.verbose_retry = true
-  # show exception that triggers a retry if verbose_retry is set to true
-  config.display_try_failure_messages = true
-
-  # set retry count and retry sleep interval to 10 seconds
-  config.default_retry_count = 5
-  config.default_sleep_interval = 10
-end
```

Comment:
Does anyone think I should port this retry configuration to minitest? I did not notice it to be needed in several dozen local test runs, so I omitted it in hopes that it was added at a time when the service was having temporary issues.

Reply:
I have a bias towards retries in tests but idk if the client handles that already

Reply:
OK if we add it later when we actually experience service failures? I will be happy to.